### DISCUSSION TITLE
공용 해시태그 컴포넌트의 범주

### DISCUSSION THREAD

**Discussion by dev-hamster** [linkarchive/Front-End#235] [General]
링크추가, 해시태그 관리의 해시태그 컴포넌트는 버튼의 기능에 가깝다고 생각해서 [이슈](https://github.com/linkarchive/Front-End/issues/230)에서 제외했는데 어떻게 생각하시나요?_? 

링크추가 > 해시태그 컴포넌트
<img width="240" alt="스크린샷 2023-08-07 오전 2 06 50" src="https://github.com/linkarchive/Front-End/assets/123740296/5b518b80-ae8f-4a1b-9aaa-df3e8d696d8c">

해시태그 관리 > 해시태그 컴포넌트
<img width="240" alt="스크린샷 2023-08-07 오전 2 07 00" src="https://github.com/linkarchive/Front-End/assets/123740296/2e5c3190-b9e9-43f9-b213-3e93d5cee69d">

**Comment by 1yoouoo** [comment]
https://github.com/linkarchive/Front-End/pull/242
